app/views/errors/forbidden.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class='main'>
+  <h1>Woops!</h1>
+  <p>You do not have access to this page.</p>
+  <p>  Are you <%= link_to 'signed in', '/auth/google_oauth2' %>?</p>
```

Comment:
I feel like a tool for bringing this up but I think the question mark should be with the signed in. :nervous_smiley: